about.html

```diff
@@ -37,7 +37,7 @@
           <a href="">&times;</a>
           <a href="index.html" class="close-menu">Home</a>
           <a href="about.html" class="close-menu">About Us</a>
-          <a href="#" class="close-menu">Inquirires</a>
+          <a href="" class="close-menu">Inquirires</a>
           <a href="#" class="close-menu">Partner</a>
        </div>
        
```

Comment:
- [ ] It should be possible to navigate between the pages, kindly make sure to add a link to the navbar in the `about` page that leads to the `home` page. You can change the About link in the `about.html` to the Home link instead so it's possible to navigate back to the Home page